src/OVAL/probes/oval_fts.c

```diff
@@ -577,35 +446,15 @@ static int process_pattern_match(const char *path, pcre **regex_out)
 		}
 		*/
 		break;
-	case PCRE_ERROR_BADPARTIAL:
```

Comment:
We don't need that case anymore. Any recent version of the library does not have limits on partial matches. This is just a dead code block now.